Makefile

```diff
@@ -43,27 +43,21 @@ fmt: ## Run go fmt against code.
 vet: ## Run go vet against code.
 	go vet ./...
 
-godeps-update:
+godeps-update: ## Run go mod tidy and go mod vendor.
 	go mod tidy && go mod vendor
 
-test: manifests generate fmt vet ## Run tests.
-	mkdir -p ${ENVTEST_ASSETS_DIR}
-	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo \
-		${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.8.3/hack/setup-envtest.sh
-	source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); \
-		go test -coverprofile cover.out `go list ./... | grep -v "e2e"`
+test-setup: generate fmt vet godeps-update ## Run setup targets for tests
```

Comment:
We should also add `manifests` as it was present in the `test`

Reply:
This is specifically for the golang unit tests. Do we really need to generate YAML manifests for them? It's fine if yes, I'm just not aware of where we need that.

Reply:
`manifests` are mainly used in the bundle as far as know and we are calling it specifically in the bundle at many other places. So we should be good even if we do not add it.